cylc/flow/scheduler.py

```diff
@@ -360,6 +364,9 @@ async def initialise(self):
         * Initialise managers.
 
         """
+        if self.is_restart:
+            self.load_workflow_params_and_tmpl_vars()
+
```

Comment:
I think moving this will cause problems - and it looks like it's resulted in several test failures

Reply:
That would not be surprising, there is a lot of order sensitivity here.